MySQL.Data/src/MySqlCommand.cs

```diff
@@ -760,7 +760,7 @@ internal async Task<MySqlDataReader> ExecuteReaderAsync(CommandBehavior behavior
 
       // Tell whoever is listening that we have started out command
 #if NET5_0_OR_GREATER
-      CurrentActivity = MySQLActivitySource.CommandStart(this);
+      CurrentActivity = MySQLActivitySource.CommandStart(this, sql);
```

Comment:
While I wanted to avoid passing in the `sql` variable and use the `MySqlCommand`, it seemed most appropriate - `db.statement` reflects [the statement to be executed by the database](https://opentelemetry.io/docs/specs/semconv/database/database-spans/#call-level-attributes). The `sql` variable is most closely that.